src/actions/errors.js

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+//
```

Comment:
super nit: excess empty comment line 😄

Reply:
I removed it and also added a comment explaining the purpose for this file.